tests/reftests/init.test

```diff
@@ -116,36 +116,23 @@ Configuring from ${BASEDIR}/opamrc and then from built-in defaults.
 <><> Fetching repository information ><><><><><><><><><><><><><><><><><><><><><>
 [default] Initialised
 
-<><> Creating initial switch 'default' (invariant ["ocaml" {>= "4.05.0"}] - initially with ocaml-system)
```

Comment:
This test needs to be fixed with the new behaviour (add -c ocaml-system), and another one added for the change.